Networking/NetworkingTests/Mapper/ProductAttributeTermListMapperTests.swift

```diff
@@ -0,0 +1,35 @@
+import XCTest
+@testable import Networking
+
+final class ProductAttributeTermListMapperTests: XCTestCase {
+    /// Dummy Site ID.
+    ///
+    private let dummySiteID: Int64 = 33334444
+
+    func test_productAttributeTerm_fields_are_correctly_mapped() throws {
+        let terms = try mapLoadAllProductAttributeTermsResponse()
+        XCTAssertEqual(terms.count, 3)
+
+        let secondTerm = terms[1]
+        XCTAssertEqual(secondTerm.siteID, dummySiteID)
+        XCTAssertEqual(secondTerm.termID, 27)
+        XCTAssertEqual(secondTerm.name, "Medium")
+        XCTAssertEqual(secondTerm.slug, "medium")
+        XCTAssertEqual(secondTerm.count, 1)
```

Comment:
nit: does it make sense to assert on the whole term using the default `Equatable` implementation, since this entity doesn't seem overly large (some entities like `Product` or `Order` has tens of attributes)?